zebra-chain/src/transaction/shielded_data.rs

```diff
@@ -65,10 +65,23 @@ impl Arbitrary for Spend {
     type Strategy = BoxedStrategy<Self>;
 }
 
+#[derive(Deserialize, Serialize)]
+#[serde(remote = "jubjub::AffinePoint")]
+struct AffinePoint {
```

Comment:
Could this be called something that indicates it's a Serde helper, like `AffinePointHelper`?

Reply:
Also `jubjub::AffinePoint` is used in several `sapling` key types, if those ever get serialized (for wallet-related stuff) having the same serde helper used for all instances would be good.

Reply:
How about I make a `serde_helper` module and move all of these types into that centralized location.